### PR TITLE
Update Manage Jenkins icons

### PR DESCRIPTION
**Pull request by janfaracik**
Settings UI

This is the first of several more focused PRs to #6186, this PR:

* Updates the Manage Jenkins page to use Jenkins Symbols 
* Updates the Manage Jenkins control's hover/focus states to be consistent with the new form components in #5923 
* The plugins icon now shows how many updates are available 
* Updates the nodes icons

The majority of files changed in this PR are:
* Adding symbols
* Updating Manage Jenkins actions to use those symbols
* Removing translation files for the older updates message

### Proposed changelog entries

* Update Manage Jenkins icons and controls 
* The plugins icon now shows how many updates are available 

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

## Test this pull request

```command
docker run --rm -ti -p 8080:8080 -e ID=6273 -e MERGE_WITH=master jenkins/core-pr-tester
```

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Comment by daniel-beck** [comment]
What's with the monochrome look? I look at this and it just seems to make it more difficult to find options because color as a tool is just gone.

**Comment by NotMyFault** [comment]
I agree with Daniel, though the monochrome theme doesn't look bad per se, it just makes it harder to differentiate between the icons and each category.

> Could we use different colors for each icon perhaps?

To pick this idea up, how feasible is it to have different color gradients for each category?

**Comment by timja** [comment]
The credentials icons look skewed:
image

(Either a PR to the credentials plugin to switch it over or a PR here should be done I think)

I checked a few plugins that provide `ManagementLinks` and they look fine

- Docker
- JCasC
- Monitoring

**Review by uhafner — APPROVED** [review]
I like the new design! Some plugins need to be updated but that can be done step by step.

The icons in the side panel all use blue while the other icons use black. I am not sure if that makes sense. For me the black ones look better but this might be a matter of preference. 

I would not use different colors in the icons, I think the icons are all distinguishable very well. And with a single color it looks much better.

Two things (can be added in a future PR):
- The new Node action still uses the wrong icon:
Bildschirmfoto 2022-02-14 um 11 15 19
- The not-available icon also uses a different icon (and the alignment looks wrong):
Bildschirmfoto 2022-02-14 um 11 22 06

**Review by fqueiruga — APPROVED** [review]
Looks lovely!

**Review by NotMyFault — APPROVED** [review]
Looks great 🎉 

I guess the credentials plugin has just a scuffed icon, docker or the config file provider plugin are looking normal.

**Comment by janfaracik** [comment]
Thanks for the comments everyone :) 

**On the icon colour**

Personally I prefer the monochrome icons, although I'm definitely not opposed to individual or group colours for them - happy to hear more thoughts on this/hear from the community.

Having the icons each be different colours might be a little tricky with how Symbols currently work (although it'd definitely be doable with some thinking), group colours would be rather simple to do 👍 

Right now the sidebar icons are blue so that they don't stand out as much from the existing Tango icons, in the future I'd like to update them all (and the sidebar design) to look something more like - 

image

**On the icon stretching**

I've pushed a small change which should fix the icon stretching issue so that PR might not be necessary @NotMyFault - thanks for the help though :)

Thanks everyone :)

**Comment by NotMyFault** [comment]
> I've pushed a small change which should fix the icon stretching issue so that PR might not be necessary @NotMyFault - thanks for the help though :)

It had bad viewpoints, can't hurt to get it sorted there too.

Do you plan to add the new ionicons in a follow up PR to the translation map?

**Review by timja — APPROVED** [review]
That fixed the skewing thanks!

looks good to me now and in dark theme

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by StefanSpieker — APPROVED** [review]

**Comment by janfaracik** [comment]
> > I've pushed a small change which should fix the icon stretching issue so that PR might not be necessary @NotMyFault - thanks for the help though :)
> 
> It had bad viewpoints, can't hurt to get it sorted there too.
> 
> Do you plan to add the new ionicons in a follow up PR to the translation map?

Yep, I'll add lots of translations to the follow up PR to this one (updating all the sidebar icons etc) :)

File: core/src/main/resources/jenkins/model/Jenkins/manage.jelly
Comment (daniel-beck) [inline core/src/main/resources/jenkins/model/Jenkins/manage.jelly:92]: This makes no sense here. Why remove the custom Jelly file?

File: core/src/main/resources/hudson/model/User/index.jelly
Comment (basil) [inline core/src/main/resources/hudson/model/User/index.jelly:31]: Causes [JENKINS-70023](https://issues.jenkins.io/browse/JENKINS-70023).

**Pull request merged**
